Source/WebKit/Platform/cocoa/WebPrivacyHelpers.mm

```diff
@@ -406,6 +406,33 @@ static RestrictedOpenerType restrictedOpenerType(WPRestrictedOpenerType type)
     return it == m_restrictedOpenerTypes.end() ? RestrictedOpenerType::Unrestricted : it->value;
 }
 
+ResourceMonitorURLsController& ResourceMonitorURLsController::singleton()
+{
+    static MainThreadNeverDestroyed<ResourceMonitorURLsController> sharedInstance;
+    return sharedInstance.get();
+}
+
+void ResourceMonitorURLsController::prepare(CompletionHandler<void(WKContentRuleList*, bool)>&& completionHandler)
+{
+    ASSERT(RunLoop::isMain());
+    if (!PAL::isWebPrivacyFrameworkAvailable() || ![PAL::getWPResourcesClass() instancesRespondToSelector:@selector(prepareResouceMonitorRulesForStore:completionHandler:)]) {
```

Comment:
Looks like the name of the method is actually `loadResouceMonitorRulesForStore:completionHandler:`.